.eleventy.js

```diff
@@ -0,0 +1,59 @@
+const makeItemLink = (slug) => `#${slug}`
+
+module.exports = function (config) {
+  // Add a filter using the Config API
+  config.addFilter('linkTarget', makeItemLink);
+
+  config.addFilter('linkIfExistsInCollection', (word, collection) => {
+    const existingDefintion = collection.find(item => item.data.title === word)
```

Comment:
I’m really sorry to do this but `definition` is spelled wrong here

Reply:
Haha. Thanks. No worries. As I said in the OP: the project would be very well founded if anyone would donate for my inability to type definition.